### PR TITLE
Create README.md

### LINKED ISSUE #2: file missing
can't find "Readme.txt" file. 
i saw the video though and it would be better if this has a readme file.
i dont understand what file does what.

### PR DESCRIPTION
**Pull request by x0samnan**
- Closes #2 and #8.
- Added guide to run locally.
- Added guide to run on a cloud instance.

### CODE REVIEW

**Comment by x0samnan** [comment]
Another PR for readme is already opened. This will be closed.

**Comment by x0samnan** [comment]
@hkirat Please, have a look at this PR. Let me know, if there are any changes to be made.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# LINKEDIN-BOT
+A linkedin-bot which automatically accepts all the connection invitations. 
+
+## Running it locally on your machine
+1) Clone this repo and add .env file with your username and password.
+2) Make sure node is installed on your system.
+3) Install chrome-driver on your system.
+4) Install selenium and dotenv using "npm i selenium dotenv"
```
Comment (laukik-target) [inline README.md:8]: Better to "cd " into the project root and run "npm i". 

This way in the future if any new dep added no need to update README

File: README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+&emsp;> export USERNAME="Replace this with your linkedin username without quotes)" <br />
+&emsp;> export PASSWORD="Replace this with your linkedin password without quotes)" <br />
+&ensp;3.b) Windows-CMD <br />
+&emsp;> setx USERNAME "Replace this with your linkedin username without quotes" <br />
+&emsp;> setx PASSWORD "Replace this with your linkedin password without quotes" <br />
+&ensp;3.c) Powershell <br />
+&emsp;> $Env:USERNAME="Replace this with your linkedin username" <br />
+&emsp;> $Env:PASSWORD="Replace this with your linkedin password" <br />
+4) Install node, chrome-driver and selenium.
+5) Run the application.
```
Comment (laukik-target) [inline README.md:25]: A better way to run on AWS is using pm2 

checkout: [](https://pm2.keymetrics.io/)

File: README.md (hunk 1 above)
Comment (x0samnan) [reply README.md:8]: @laukik-target Will change the code but since we are making a .env file in first case, I think dotenv has to be manually installed.

**Comment by x0samnan** [comment]
@hkirat Are there any more changes required or is the PR ready to be merged?

**Review by cleverhare — CHANGES_REQUESTED** [review]
It actually looks good but a sort of screenshot of the bot working fine is what missing in this readme so if you add on that it will be a nice readme

**Comment by x0samnan** [comment]
@cleverhare Thanks for the suggestion. Actually, to show the working of bot, a screenshot would not be enough so a video would be rather great. If you don't mind, can you work on that bit after this PR gets merged?

**Comment by cleverhare** [comment]
Yeah we can work together no worries

**Comment by x0samnan** [comment]
@hkirat Is this Readme PR ready to be merged or are there any changes required?

**Review by cleverhare — CHANGES_REQUESTED** [review]
Please don't share your credentials.  Add the .env file to the gitignore file

**Comment by x0samnan** [comment]
@cleverhare Is my .env pushed?

**Comment by cleverhare** [comment]
Yes it is pushed, Change the Password and do add it in gitignore file

**Comment by x0samnan** [comment]
@cleverhare I can't see it on my end. I will close this and open a new PR.

**Comment by cleverhare** [comment]
Okay

**Comment by x0samnan** [comment]
@cleverhare Refer to #11 for working on it.

**Pull request closed without merging**